gyp.js

```diff
@@ -40,16 +40,14 @@ function runGyp (opts, cb) {
       }
     }
 
-    gyp.commands[command.name](command.args, function (err) {
-      if (err) {
-        log.error(command.name + ' error')
-        log.error('stack', err.stack)
-        log.error('not ok')
-        return cb(err)
-      }
-
+    gyp.commands[command.name](command.args).then(function () {
```

Comment:
@orgads @lovell Does this change also work with other backends like node-ninja? AFAIK they are still running callback style functions.

Reply:
https://github.com/prebuild/prebuild/issues/323